src/components/tasks/card/index.tsx

```diff
@@ -12,6 +13,7 @@ import {
     COMPLETED,
     VERIFIED,
 } from '@/components/constants/beautified-task-status';
+import { DUMMY_PROFILE } from '@/components/constants/display-sections';
```

Comment:
Suggestion:

Use alias name `placeholderImageURL` for `DUMMY_PROFILE`

Snippet:

```js
import { DUMMY_PROFILE as placeHolderImageURL } from '@/components/constants/display-sections';
```

Reply:
Changes done